### DISCUSSION TITLE
VitePress博客搭建 | Goalonez

### DISCUSSION THREAD

**Discussion by Goalonez** [Goalonez/goalonez.github.io#153] [Announcements]
https://blog.goalonez.site/blog/VitePress%E5%8D%9A%E5%AE%A2%E6%90%AD%E5%BB%BA.html 

 Goalonez Blog

**Comment by SalamancaLaluo** [comment]
踩坑记录一下，首页这里原文是
编辑./docs/index.md，其实是编辑./index.md  ,我试了很多次./docs/index.md都失败了

**Comment by SalamancaLaluo** [comment]
踩坑原因是init的时候没有指定目录，我哭了

**Comment by SalamancaLaluo** [comment]
感恩大佬出的教程
